report/src/main/kotlin/com/hartwig/actin/report/pdf/ReportContentProvider.kt

```diff
@@ -161,47 +149,55 @@ class ReportContentProvider(private val report: Report, private val enableExtend
                 report.config.includeEligibleSOCTreatmentSummary
             }, EligibleApprovedTreatmentGenerator(report, contentWidth).takeIf {
                 report.config.includeApprovedTreatmentsInSummary
-            }) +
-                primaryCohortsGenerators + otherCohortGenerators +
-                listOfNotNull(
-                    localTrialGenerator.takeIf { report.config.includeExternalTrialsInSummary },
-                    nonLocalTrialGenerator.takeIf { report.config.includeExternalTrialsInSummary },
-                    IneligibleActinTrialsGenerator.forOpenCohorts(
-                        cohorts, report.requestingHospital, contentWidth, enableExtendedMode
-                    ).takeIf {
-                        report.config.includeIneligibleTrialsInSummary
-                    })
+            }) + trialTableGenerators
     }
 
-    private fun getGeneratorsForSource(
-        cohorts: List<InterpretedCohort>,
+    private fun getGenerators(
+        interpretedCohorts: List<InterpretedCohort>,
+        externalTrialSummary: SummarizedExternalTrials,
         requestingSource: TrialSource?,
-        source: TrialSource? = requestingSource,
-        contentWidth: Float,
-        includeLocation: Boolean = false,
-    ): List<EligibleActinTrialsGenerator?> {
-        val openCohortsWithSlotsGenerator = EligibleActinTrialsGenerator.forOpenCohorts(
-            cohorts, source?.description, contentWidth, slotsAvailable = true, includeLocation = includeLocation
+        contentWidth: Float
+    ): List<TrialTableGenerator?> {
+        val homeCountry = report.config.countryOfReference
```

Comment:
Wouldn't define this val here. Related, `homeCountry` should be renamed to `countryOfReference` everywhere but can be considered a side project